Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2426,6 +2465,58 @@ AODProducerWorkflowDPL::TrackExtraInfo AODProducerWorkflowDPL::processBarrelTrac
   return extraInfoHolder;
 }
 
+AODProducerWorkflowDPL::TrackQA AODProducerWorkflowDPL::processBarrelTrackQA(int collisionID, std::uint64_t collisionBC, GIndex trackIndex,
+                                                                             const o2::globaltracking::RecoContainer& data, const std::map<uint64_t, int>& bcsMap)
+{
+  TrackQA trackQAHolder;
+  auto contributorsGID = data.getSingleDetectorRefs(trackIndex);
+  const auto& trackPar = data.getTrackParam(trackIndex);
+  auto src = trackIndex.getSource();
```

Comment:
why did you leave this? This is not used.

Reply:
2474-removed in next commit.
2473. we are using trackPar to get momenta later  - I assume this is momenta of combined track

Reply:
fixed